src/App/Fossa/FossaAPIV1.hs

```diff
@@ -579,3 +580,20 @@ resolveProjectDependencies apiOpts locator = fossaReq $ do
   (dependencies :: [ResolvedDependency]) <- responseBody <$> req GET (resolveProjectDependenciesEndpoint baseUrl locator) NoReqBody jsonResponse opts
 
   pure $ map unwrapResolvedDependency dependencies
+
+----------
+
+newtype FeatureFlagEnabledBody = FeatureFlagEnabledBody {unFeatureFlagEnabled :: Bool}
+
+instance FromJSON FeatureFlagEnabledBody where
+  parseJSON = withObject "FeatureFlagEnabledBody" $ \obj -> do
```

Comment:
[optional nit] Maybe way too precautionary - I presume there is contract test on backend to ensure, this FF doesn't get removed from underneath us?